components/templates/NewsPage/Contacts/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { ReleaseContact } from "@/lib/api/noirlab/codegen";
+import Stack from "@rubin-epo/epo-react-lib/Stack";
+import * as Styled from "../styles";
+
+interface ContactsProps {
+  contacts: Array<ReleaseContact> | (string | undefined);
+}
+
+const Contacts: FunctionComponent<ContactsProps> = ({ contacts = [] }) => {
```

Comment:
revamp the `Contacts` component, announcements send their contacts as an HTML chunk instead of an array. Display will be slightly different between press releases and announcements because of this.